tests/performance/specs/shopper/checkout.js

```diff
@@ -0,0 +1,355 @@
+          origin: `${base_url}`,
+          "sec-fetch-dest": "empty",
+          "sec-fetch-mode": "cors",
+          "sec-fetch-site": "same-origin",
+          "x-requested-with": "XMLHttpRequest",
+          "sec-ch-ua":
+            '"Chromium";v="88", "Google Chrome";v="88", ";Not A Brand";v="99"',
+          "sec-ch-ua-mobile": "?0",
+        },
+      }
+    );
+    addToCartTrend.add(response.timings.duration);
+  });
+
+  sleep(randomIntBetween(`${think_time_min}`, `${think_time_max}`));
+
+  group("View Cart", function () {
+    response = http.get(`${base_url}/cart`, {
+      headers: {
+        accept:
+          "text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*//*;q=0.8,application/signed-exchange;v=b3;q=0.9",
+        "accept-encoding": "gzip, deflate, br",
+        "accept-language": "en-US,en;q=0.9",
+        connection: "keep-alive",
+        host: `${base_host}`,
+        "sec-fetch-dest": "document",
+        "sec-fetch-mode": "navigate",
+        "sec-fetch-site": "same-origin",
+        "sec-fetch-user": "?1",
+        "upgrade-insecure-requests": "1",
+        "sec-ch-ua":
+          '"Chromium";v="88", "Google Chrome";v="88", ";Not A Brand";v="99"',
+        "sec-ch-ua-mobile": "?0",
+      },
+    });
+    viewCartTrend.add(response.timings.duration);
+    check(response, {
+      "body does not contain 'your cart is currently empty'": response =>
+        !response.body.includes("Your cart is currently empty."),
+    });
+  });
+
+  sleep(randomIntBetween(`${think_time_min}`, `${think_time_max}`));
+
+  group("Proceed to checkout", function () {
+    response = http.get(`${base_url}/checkout`, {
+      headers: {
+        accept:
+          "text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*//*;q=0.8,application/signed-exchange;v=b3;q=0.9",
+        "accept-encoding": "gzip, deflate, br",
+        "accept-language": "en-US,en;q=0.9",
+        connection: "keep-alive",
+        host: `${base_host}`,
+        "sec-fetch-dest": "document",
+        "sec-fetch-mode": "navigate",
+        "sec-fetch-site": "same-origin",
+        "sec-fetch-user": "?1",
+        "upgrade-insecure-requests": "1",
+        "sec-ch-ua":
+          '"Chromium";v="88", "Google Chrome";v="88", ";Not A Brand";v="99"',
+        "sec-ch-ua-mobile": "?0",
+      },
+    });
+    proceedCheckoutTrend.add(response.timings.duration);
+  });
+
+  sleep(randomIntBetween(`${think_time_min}`, `${think_time_max}`));
+
+  vars["woocommerce-process-checkout-nonce"] = response
+    .html()
+    .find("input[name=woocommerce-process-checkout-nonce]")
+    .first()
+    .attr("value");
+
+  group("Place Order", function () {
+    response = http.post(
+      `${base_url}/?wc-ajax=update_order_review`,
+      {
+        security: "ae0d006f28", //TODO: Correlate this value from update_order_review_nonce on ${base_url}/checkout. Test works with it hard coded for now.
```

Comment:
I didn't need to modify these but was still able to place orders. Can you tell me more about why this is needed? Can we use some other string to verify if the request was successful?

Reply:
I discovered although the orders were placed successfully without modifying this it was causing the request to response with 403 Forbidden.
I have managed to correlate this from the `update_order_review_nonce` and now it works much better with 200 response.